evap/evaluation/tests/test_auth.py

```diff
@@ -144,6 +144,59 @@ def test_oidc_login(self):
         self.assertIn("Logout", page.body.decode())
         self.assertEqual(page.context["user"], user)
 
+    @override_settings(ACTIVATE_OPEN_ID_LOGIN=True)
+    @patch.multiple(
+        auth.OIDCAuthenticationBackend,
+        get_userinfo=DEFAULT,
+        get_token=DEFAULT,
+        store_tokens=DEFAULT,
+        verify_token=DEFAULT,
+    )
+    def test_oidc_email_transitions(self, get_userinfo, verify_token, **_other_mocks):
```

Comment:
I was wondering what happens if there already exists both an `institution.example.com` and `student.institution.example.com` login. This works currently, because we `yield email` before `settings.OIDC_EMAIL_TRANSITIONS.get(domain)`. In fact if the yield order was changed and you'd try to log in with such a user using the `institution.example.com` email, you get an IntegretyError as the accounts are not merged. Applies for example to `vincenzo.boston@institution.example.com`.

Maybe we should add a test that the "correct" account is used when both accounts exist? In a follow up ticket (or even here?) we could directly merge the two user accounts